cmd/apiserver-boot/boot/init_repo/repo.go

```diff
@@ -268,15 +288,13 @@ type buildTemplateArguments struct {
 }
 
 var buildTemplate = `
-# gazelle:proto disable
-# gazelle:exclude vendor
-load("@io_bazel_rules_go//go:def.bzl", "gazelle")
+load("@bazel_gazelle//:def.bzl", "gazelle")
 
+# gazelle:proto disable_global
+# gazelle:prefix {{.Repo}}
 gazelle(
     name = "gazelle",
     command = "fix",
-    prefix = "{{.Repo}}",
-    external = "vendored",
     args = [
         "-build_file_name",
         "BUILD,BUILD.bazel",
```

Comment:
Actually, this should be `BUILD.bazel` not `BUILD` with recommendation, since `BUILD` may have confusion with other tools' files which also name them as `build`. I also want to change this `BUILD,BUILD.bazel` to `BUILD.bazel` in apiserver-builder-alpha's root `BUILD.bazel` in the future.

Reply:
renamed BUILD.bazel to BUILD